app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -140,7 +140,9 @@ register_katello_host(){
 <%= "      --data 'host[organization_id]=#{@organization.id}' \\\n" if @organization -%>
 <%= "      --data 'host[location_id]=#{@location.id}' \\\n" if @location -%>
 <%= "      --data 'host[hostgroup_id]=#{@hostgroup.id}' \\\n" if @hostgroup -%>
+<%= "      --data 'host[operatingsystem_id]=#{@operatingsystem.id}' \\\n" if @operatingsystem -%>
 <%= "      --data 'host[lifecycle_environment_id]=#{@lifecycle_environment_id}' \\\n" if @lifecycle_environment_id.present? -%>
+<%= "      --data host[interfaces_attributes][0][identifier]=#{shell_escape(@remote_execution_interface)} \\\n" if @remote_execution_interface.present? -%>
```

Comment:
Why escape this instead of quoting?
```suggestion
<%= "      --data 'host[interfaces_attributes][0][identifier]=#{@remote_execution_interface}' \\\n" if @remote_execution_interface.present? -%>
```

I suspect you copied it from 2 lines below, but now I wonder why that was the way it was.

Reply:
I simply copied what register_host() did. I also asked myself why it was coded this way but simply assumed the original author probably has probably thought this through enough and I wanted to avoid breaking any weird escaping edge-cases by accident.

Reply:
Looking at the PR that intoduced this line and the linked issue, this seems to indeed stem from some edge-cases with escaping certain characters, specifically single quotes:
https://projects.theforeman.org/issues/33033
https://github.com/theforeman/foreman/pull/8687

I have no idea if single quotes are a valid part of an interface name

Reply:
@ekohl do you have any more thoughts on this? With the reasoning from the PR that originally introduced this syntax, I would opt towards leaving shell_escape in just to be save.